### ISSUE TITLE
Weird Visual Anomalies When Using Headings

### ISSUE THREAD

**Issue by madranet** [asciidoctor/asciidoctor-vscode#817]
VSCODIUM VERSION:
```
Version: 1.84.0
Release: 23306
Commit: 695a9146b1347633e608f9f166c81b5b2843b492
Date: 2023-11-02T09:32:48.179Z (4 days ago)
Electron: 25.9.2
ElectronBuildId: undefined
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Darwin x64 20.6.0
```
PLUGIN VERSION:
```
AsciiDoc v3.1.7
```

This has just started happening today [although I haven't used the plugin for a couple of weeks, so it may have broken earlier]. But was working perfectly before.

Whenever I try and type headings, the editor visually glitches really badly. It's almost like it's soft-wrapping on top of itself. Here's a quick demo of what I mean. As you can see, from when I switch to the Preview pane, the text actually written to the file seems correct. But the visual anomalies are making it really confusing to work with.

Any ideas, anyone?

https://github.com/asciidoctor/asciidoctor-vscode/assets/669658/d0d2ce6f-16b5-4b82-b5e8-a3beb9aa3247

**Comment by ggrossetie** [comment]
That's really weird indeed 😱🤔
Do you reproduce this issue using the Markdown extension?

I don't think it's related to the AsciiDoc extension since I cannot reproduce it and it seems like a visual glitch of VS Codium.

You might want to try to reinstall VS Codium or switch to a previous version of the AsciiDoc extension.

**Comment by madranet** [comment]
I think I've found the culprit.

I had another thought that; what was happening almost looked like when Github CoPilot tries to autocomplete code. So I looked through my extensions and found this...

![SS 2023-11-06 at 12 51 04](https://github.com/asciidoctor/asciidoctor-vscode/assets/669658/2514d2e3-3da3-41eb-bc79-fa5e2a796e9e)

Once I disabled that extension, Asciidoctor-VScode started behaving itself again.

Sorry for the false alarm!
